src/ardrive.ts

```diff
@@ -39,23 +45,64 @@ export class ArDriveAnonymous extends ArDriveType {
 	}
 }
 
+// TODO: ArDrive should accept App-Name and App-Version tags from constructor?
+const commTipMetaTags: GQLTagInterface[] = [
+	{ name: 'App-Name', value: 'ArDrive-CLI' },
+	{ name: 'App-Version', value: '2.0' },
+	{ name: 'Trx-Type', value: 'Community-Tip' }
```

Comment:
Is this a new tag that we'll be adding to community tips? Here's how they're constructed on web;
```
feeTx = await _arweave.client.transactions.prepare(
        Transaction(
          target: await _pst.getWeightedPstHolder(),
          quantity: pstFee,
        ),
        profile.wallet,
      )
        ..addApplicationTags()
        ..addTag('Type', 'fee')
        ..addTag(TipType.tagName, TipType.dataUpload);
      await feeTx!.sign(profile.wallet);
```